decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -51,83 +51,6 @@
     official_url { Faker::Internet.url }
   end
 
-  factory :participatory_process, class: Decidim::ParticipatoryProcess do
```

Comment:
Same as before, these are lelfovers from a previous version where categories & moderations were only supported by participatory processes. Now they are fully polymorphic so everything related to those can stay in core. I'll review it!